app/templates/index.html

```diff
@@ -7,11 +7,19 @@
     <div class="panel panel--error">
       <div class="panel__header">
         <h1 class="panel__title venus">
-          {% trans count=messages|length %}
-          This page has {{ count }} error
-          {% pluralize %}
-          This page has {{ count }} errors
-          {% endtrans %}
+          {% for message in messages %}
+            {% if message.clickable %}
+              {% trans count=messages|length %}
+              This page has an error
```

Comment:
If there are multiple messages, "This page..." will be displayed for each. Intended?